manifest.json

```
@@ -1,5 +1,5 @@
 {
-	"manifest_version": 2,
+	"manifest_version": 3,
 	"name": "PhantomBuster",
 	"version": "1.1.6.8",
```

plz bump the version, at least a patch I think (so you have to update `beta-updates.json`, `manifest.beta.json` and `package.json` files too
```suggestion
	"version": "1.1.7.0",
```